analysed-packages/openvpn/version-2.5.6/openvpn-2.5.6-SPDX2TV.spdx

```diff
@@ -0,0 +1,7244 @@
+FileChecksum: SHA256: 7b3b653a3b256dec2d25ffaa12ce557dcade2b8fe57c969b3e53f025a5a806a4
+FileChecksum: MD5: 53e627f03d2b741a129da926c51ca8aa
+LicenseConcluded: LicenseRef-GPL-2.0-or-later-WITH-Autoconf-exception-generic
+LicenseInfoInFile: LicenseRef-GPL-2.0-or-later
+LicenseInfoInFile: LicenseRef-Autoconf-exception
+LicenseInfoInFile: LicenseRef-Autoconf-exception-generic
+FileCopyrightText: <text> Copyright (C) 1996-2020 Free Software Foundation, Inc. Originally written by Francois Pinard <pinard@iro.umontreal.ca>, 1996. </text>
+
+##File
+
+FileName: openvpn-2.5.6.tar.gz/openvpn-2.5.6.tar/openvpn-2.5.6/ltmain.sh
+SPDXID: SPDXRef-item546541
+FileChecksum: SHA1: 031f7e2297cd59a8861bf9854bfadf81dc3d6d8b
+FileChecksum: SHA256: f03f3f556004e494c16ef260d2e4fb275f991016824b015452fef31d6f48a77b
+FileChecksum: MD5: 4bf469ef4424ddd4b651e800fdf860ce
+LicenseConcluded: LicenseRef-GPL-2.0-or-later-WITH-Libtool-exception
+LicenseInfoInFile: LicenseRef-GPL-3.0-or-later
+LicenseInfoInFile: LicenseRef-Libtool-exception
+FileCopyrightText: <text> Copyright (C) 1996-2015 Free Software Foundation, Inc.
+Copyright (C) 2004-2015 Free Software Foundation, Inc.
+Copyright (C) 2010-2015 Free Software Foundation, Inc. </text>
+
+##File
+
+FileName: openvpn-2.5.6.tar.gz/openvpn-2.5.6.tar/openvpn-2.5.6/install-sh
+SPDXID: SPDXRef-item546547
+FileChecksum: SHA1: 297b46f08fba07c0484ef540f9f3047d7b326cc8
+FileChecksum: SHA256: c2fd71953b47c19c62db69cbf739504aca8b47a40dbdc8c83bbc4cc7664a1779
+FileChecksum: MD5: d651f428c7739cf424b31c05dfdbed04
+LicenseConcluded: LicenseRef-Public-domain AND LicenseRef-X11
+LicenseComments: <text> The information in the file is:
+
+FSF changes to this file are in the public domain.
+
+Thus, we concluded Public-domain. </text>
+LicenseInfoInFile: LicenseRef-X11
+LicenseInfoInFile: LicenseRef-MIT
+LicenseInfoInFile: LicenseRef-Public-domain
+FileCopyrightText: <text> Copyright (C) 1994 X Consortium </text>
+
+##File
+
+FileName: openvpn-2.5.6.tar.gz/openvpn-2.5.6.tar/openvpn-2.5.6/depcomp
+SPDXID: SPDXRef-item546998
+FileChecksum: SHA1: d429169aeaa44fd5b17f40defacdb9d6767e389a
+FileChecksum: SHA256: b31b449b11f40156b18d2e2a989d78ca3f89be0b96ca0a160455b3299b82d2e6
+FileChecksum: MD5: 91b287ce6c245893a75bd18dc53fb07a
+LicenseConcluded: LicenseRef-GPL-2.0-or-later-WITH-Autoconf-exception-generic
+LicenseInfoInFile: LicenseRef-GPL-2.0-or-later
+LicenseInfoInFile: LicenseRef-Autoconf-exception
+LicenseInfoInFile: LicenseRef-Autoconf-exception-generic
+FileCopyrightText: <text> Copyright (C) 1999-2020 Free Software Foundation, Inc. </text>
+
+##File
+
+FileName: openvpn-2.5.6.tar.gz/openvpn-2.5.6.tar/openvpn-2.5.6/configure
+SPDXID: SPDXRef-item546546
+FileChecksum: SHA1: e583d9d2bf45048ad9f5e849e032b72406df5b97
+FileChecksum: SHA256: 4467f3b275e549a156971fb652aabc8ceec3d8d8f53a414f5b577ceb0fc7c513
+FileChecksum: MD5: a8e8453569074e4d4093b97ef5df41c7
+LicenseConcluded: LicenseRef-FSFUL AND LicenseRef-GPL-2.0-or-later-WITH-Libtool-exception
```

Comment:
Can you please add:
This config.status script is free software; the Free Software Foundation
gives unlimited permission to copy, distribute and modify it.

to the text of LicenseRef-FSFUL